hack/bazel.sh

```diff
@@ -20,6 +20,20 @@ set -o errexit
 set -o pipefail
 
 code=0
+# Remove retries after
+# https://github.com/bazelbuild/bazel/issues/12599
+BAZEL_SYNC_PLEASE='true'
```

Comment:
forcing on test in this PR temporarily.
final PR would opt-in specific jobs to this initially (test-infra) since it's used in multiple places